README.md

```diff
@@ -132,6 +156,39 @@ TectonTriggerOperator(
 )
 ```
 
+### Feature Table Ingestion Materialization Job Submission
+```python
+import pandas as pd
+from datetime import datetime
+
+from airflow_tecton.operators.tecton_feature_table_trigger_operator import TectonFeatureTableTriggerOperator
+
+WORKSPACE = "prod"
+FEATURE_VIEW = "df_users"
+
+
+def generate_df():
```

Comment:
i think we should work out an example that uses one of the airflow context vars (probably start_time)

Reply:
sure